lib/client/api.go

```diff
@@ -430,7 +430,21 @@ func (p *ProfileStatus) KeyPath() string {
 //
 // It's kept in <profile-dir>/keys/<proxy>/<user>-db/<cluster>/<name>-x509.pem
 func (p *ProfileStatus) DatabaseCertPath(name string) string {
```

Comment:
Is this method still used by anything? If not, let's remove it.

Reply:
it's referenced a few other places in `tool/tsh`, but i can switch all of them and remove this one